spec/requests/user/user_settings_spec.rb

```diff
@@ -122,28 +124,38 @@
         expect(response).to have_http_status(:ok)
       end
 
-      it "shows the 'Remove OAuth' section if a user has multiple enabled identities" do
+      it "shows the description if a user has multiple enabled identities" do
         allow(Authentication::Providers).to receive(:enabled).and_return(Authentication::Providers.available)
         providers = Authentication::Providers.available.first(2)
         allow(user).to receive(:identities).and_return(user.identities.where(provider: providers))
 
         get user_settings_path(tab: "account")
-        expect(response.body).to include(remove_oauth_section)
```

Comment:
As I see `remove_oauth_section` is still there, so why remove it?

Reply:
because I think It's more important to check the correct description is displayed than a section title and no need to check both the description and title because of a bit redundant.